YouTube Livestream Botter.py

```diff
@@ -20,14 +23,17 @@
 
 print(intro)
 
-if platform.system() == "Windows": #checking OS
+if platform.system() == "Windows":  # checking OS
     clear = "cls"
 else:
     clear = "clear"
 
 token = input("ID?\n")
-url= "https://m.youtube.com/watch?v=" + token
-url2 = "https://s.youtube.com/api/stats/watchtime?ns=yt&el=detailpage&cpn=AiWtf2fjIwVS2MeQ&docid=" + token + "&ver=2&cmt=7926.045&ei=igGSXce3IM2NgAfasp-ABg&fmt=133&fs=0&rt=1096.007&of=L_224b5BokWsQ5UWgAws_w&euri&lact=2837&live=dvr&cl=271684942&state=playing&vm=CAEQABgEKhhJc0gwZ2w0QmFfbTBWSXlWNm9ITmRRPT0&volume=100&c=MWEB&cver=2.20190928.07.00&cplayer=UNIPLAYER&cbrand=apple&cbr=Safari%20Mobile&cbrver=12.1.15E148&cmodel=iphone&cos=iPhone&cosver=12_2&cplatform%20=MOBILE&delay=5&hl=ru_RU&cr=DE&rtn=1396&afmt=140&lio=1556394045.182&idpj=&ldpj=&rti=1096&muted=0&st=7626.045&et=7926.045 "
+url = "https://m.youtube.com/watch?v=" + token
+url2 = "https://s.youtube.com/api/stats/watchtime?ns=yt&el=detailpage&cpn=AiWtf2fjIwVS2MeQ&docid=" + token + \
```

Comment:
`+ \` will throw error, what is that

Reply:
If you want to add an \ at the end of the string you would need to do it like so: `+ "\"` , but then doing this there is a possibility of it breaking the script when changing anything to the API call.